src/main/java/uk/gov/hmcts/reform/slc/services/steps/sftpupload/Report.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.slc.services.steps.sftpupload;
+
+public class Report {
+
+    public final String path;
```

Comment:
Could you please let me know if this Is `path` or `filename`?

Reply:
it's a path